reactor-core/src/main/java/reactor/util/Loggers.java

```diff
@@ -461,25 +470,29 @@ public Logger apply(String name) {
 	 */
 	static final class ConsoleLogger implements Logger {
 
-		private final String name;
+		private final ConsoleLoggerKey identifier;
 		private final PrintStream err;
 		private final PrintStream log;
 		private final boolean verbose;
 
-		ConsoleLogger(String name, PrintStream log, PrintStream err, boolean verbose) {
-			this.name = name;
+		ConsoleLogger(ConsoleLoggerKey identifier, PrintStream log, PrintStream err) {
+			this.identifier = identifier;
 			this.log = log;
 			this.err = err;
-			this.verbose = verbose;
+			this.verbose = identifier.verbose;
```

Comment:
why duplicate the verbose field? local field could be removed in favor of `this.identifier.verbose` wdyt?

Reply:
It was just a commodity to avoid changing all `if (verbose)` with `if (identifier.verbose)`.

But both solutions work for me. If you confirm you want it replaced, I will amend my commit to do the switch.

Reply:
yeah I think it makes sense to have only one field. please just add a new commit to the branch. I'll squash everything when merging anyway (but I like the effort to have meaningful small commits in the PR branch 👍 )